sql/core/src/test/resources/sql-tests/results/transform.sql.out

```diff
@@ -202,9 +202,9 @@ FROM t
 -- !query schema
 struct<a:string,b:string,c:string,d:string>
 -- !query output
-1	true	Spark SQL	null
-2	false	Spark SQL	null
-3	true	Spark SQL	null
+1	true	Spark SQL	NULL
```

Comment:
isn't the default value `\N`?

Reply:
> isn't the default value `\N`?

it's null,  when convert to input string, it will be convert to `\N`, when handle output it's converted to null, then in final result it become `NULL`